Registry/RepositoryRegistry.php

```diff
@@ -46,6 +46,27 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
+    public function names()
```

Comment:
if there is a use case to know all names, i find it okay to have this method. is the names the same as $repositoryName in the `get` method?